tests/execution-spec-tests/get_execution_spec_tests.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+# requires jq
+# sudo apt install jq
+
+# The following two artifacts are intended for consumption by clients:
+# - fixtures.tar.gz: Generated up to the last deployed fork.
+# - fixtures_develop.tar.gz: Generated up to and including the latest dev fork.
+# As of March 2024, dev is Prague, deployed is Cancun.
+
+ARTIFACT="fixtures_develop.tar.gz"
+TARGET_DIR="fixtures"
+
+OWNER="ethereum"
+REPO="execution-spec-tests"
+VERSION="v4.2.0" #"latest"
+DOWNLOAD_URL="https://github.com/$OWNER/$REPO/releases/download/$VERSION/$ARTIFACT"
```

Comment:
Maybe we should source the `VERSION` from its own file directly, that way it's easy to update it in place automatically if we want later. But this is fine for now.

Reply:
Done! I added a new `VERSION` file and a resolution order:
1. `FIXTURE_VERSION` (env override)
2. `VERSION` file
3. ` latest` string literal if the previous two are not defined or the file does not exist.